New Unity Project/Assets/Scripts/UI/UIToolTip.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+public class UITooltip : MonoBehaviour
```

Comment:
El nombre de la clase no coincide con el nombre del script (T de tip en mayuscula/minuscula). Esto hace que salten excepciones de tipo `NullReferenceException` cuando se llama a `instate`.
```suggestion
public class UIToolTip : MonoBehaviour
```

Reply:
Corregido, se Coloco T mayus como se viene utilizando en la referencia de objetos.